continuous_integration/environment-mindeps-optional.yaml

```diff
@@ -0,0 +1,58 @@
+name: test-environment
+channels:
+  - conda-forge
+dependencies:
+  # required dependencies
+  - packaging=20.0
+  - python=3.8
+  - pyyaml=5.3.1
+  - click=8.0  # min version required by distributed
+  - cloudpickle=1.5.0  # min version required by distributed
+  - partd=1.2.0
+  - fsspec=2021.09.0
+  - importlib-metadata=4.13.0
+  - toolz=0.10.0  # min version required by distributed
+  # optional dependencies pulled in by pip install dask[diagnostics]
+  - bokeh=2.4.2
+  - jinja2=2.10.3
+  # optional dependencies pulled in by pip install dask[complete]
+  - pyarrow=7.0
+  - lz4=4.3.2
+  # optional dependencies used by dask
+  - bokeh=2.4.2
+  - cachey=0.1.1
+  - crick=0.0.3
+  - cytoolz=0.10.1
+  - dask-ml=1.4.0
+  - fastavro=0.22.6
+  - fastparquet=0.8.2
+  - h5py=2.10.0
+  - ipycytoscape=1.0.1
+  - IPython=7.9.0
+  - jinja2=2.10.3
+  - matplotlib=3.1.1
+  - mimesis=5.3.0
+  - psutil=5.7.0
+  - python-graphviz=0.8.4
+  - python-snappy=0.5.4
+  - python-xxhash=1.4.1
+  - s3fs=2021.9.0
+  - scikit-image=0.16.2
+  - scipy=1.3.2
+  - sparse=0.12.0
+  - sqlalchemy=1.4.16
+  - tblib=1.6.0
+  - tiledb-py=0.8.1
+  - zarr=2.12.0
+  - pip
+  - pip:
+      # optional dependencies pulled in by pip install dask[distributed]
+      - git+https://github.com/dask/distributed
+      - cityhash==0.2.4
+      - mmh3==2.4
```

Comment:
Ah, I see. Sounds good. If only the author of this PR https://github.com/conda-forge/staged-recipes/pull/11057 would get around to actually finishing things up!

Reply:
Went ahead and opened https://github.com/conda-forge/staged-recipes/pull/22539 since there seem to have been a decent amount of changes to `cityhash` since the initial PR - things seem to be passing there so hopefully we can follow up on this by adding it to the conda CI envs 🤞🏽